### PR TITLE
DSYS-157 & DSYS-161: Fix tests

### PR DESCRIPTION
**Pull request by CavalcanteLucas**
**Link to tasks**
https://labcodes.atlassian.net/browse/DSYS-157
https://labcodes.atlassian.net/browse/DSYS-161

**Description**
This PR implements tests for checking full width buttons.
Also in this PR, all references to `component.states()` are removed from the test codebase. That includes 17 tests across 3 test files:
- 7 times for `Checkbox` (to `localChecked`);
- 7 times for `Toggle` (to `localValue`);
- 2 times for `PasswordInput` (to `showPassword`).
In the files mentioned above, variables named as `component` are renamed to `renderedComponent` or `mountedComponent` for an improved readability.
Lastly, in the files mentioned above, the `mount` enzyme method is preferred instead of `shallow`.

**How to test**
From the labsystem dir, run `npm test`. Check if all tests pass.

### CODE REVIEW

File: labsystem/src/Checkbox.test.js [hunk 1]
```diff
@@ -158,64 +159,111 @@ describe("Checkbox", () => {
...
+    expect(
+      mountedComponent
+        .find(Icon)
+        .find("span")
+        .find(".lab-icon")
+        .find(".lab-icon--check")
+        .find(".lab-icon--white")
+        .find(".lab-icon--small")
+    ).toHaveLength(1);
+  });
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:181]: while you're checking that the checked icon is there, you're not validating that the input is being marked as checked.
please add this check as well. you may use a similar solution, just checking for `input[checked]` instead.

File: labsystem/src/Checkbox.test.js [hunk 2]
```diff
@@ -158,64 +159,111 @@ describe("Checkbox", () => {
...
-    component = mount(
+    expect(
+      mountedComponent.find(
+        Icon,
+        ".lab-icon",
+        ".lab-icon--check",
+        ".lab-icon--white",
+        ".lab-icon--small"
+      )
+    ).toHaveLength(0);
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:201]: same thing here about the checked value.

File: labsystem/src/Checkbox.test.js [hunk 3]
```diff
@@ -158,64 +159,111 @@ describe("Checkbox", () => {
...
+
+    expect(
+      mountedComponent.find(
+        Icon,
+        ".lab-icon",
+        ".lab-icon--check",
+        ".lab-icon--white",
+        ".lab-icon--small"
+      )
+    ).toHaveLength(0);
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:217]: same here.

File: labsystem/src/Checkbox.test.js [hunk 4]
```diff
@@ -158,64 +159,111 @@ describe("Checkbox", () => {
...
-    expect(component.state().localChecked).toBe(true);
+    expect(
+      mountedComponent.find(
+        Icon,
+        ".lab-icon",
+        ".lab-icon--check",
+        ".lab-icon--white",
+        ".lab-icon--small"
+      )
+    ).toHaveLength(0);
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:251]: and here

File: labsystem/src/Checkbox.test.js [hunk 5]
```diff
@@ -158,64 +159,111 @@ describe("Checkbox", () => {
...
+    mountedComponent.find("input").at(0).simulate("change");
+    expect(
+      mountedComponent
+        .find(Icon)
+        .find("span")
+        .find(".lab-icon")
+        .find(".lab-icon--check")
+        .find(".lab-icon--white")
+        .find(".lab-icon--small")
+    ).toHaveLength(1);
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:261]: and here

File: labsystem/src/Checkbox.test.js [hunk 6]
```diff
@@ -224,12 +272,28 @@ describe("Checkbox", () => {
...
-    expect(component.state().localChecked).toBe(false);
+    expect(
+      mountedComponent.find(
+        Icon,
+        ".lab-icon",
+        ".lab-icon--check",
+        ".lab-icon--white",
+        ".lab-icon--small"
+      )
+    ).toHaveLength(0);
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:283]: :]

File: labsystem/src/Checkbox.test.js [hunk 7]
```diff
@@ -224,12 +272,28 @@ describe("Checkbox", () => {
...
-    expect(component.state().localChecked).toBe(true);
+    expect(
+      mountedComponent
+        .find(Icon)
+        .find("span")
+        .find(".lab-icon")
+        .find(".lab-icon--check")
+        .find(".lab-icon--white")
+        .find(".lab-icon--small")
+    ).toHaveLength(1);
```
Comment (lucianoratamero) [inline labsystem/src/Checkbox.test.js:296]: ;)

**Review by lucianoratamero — CHANGES_REQUESTED** [review]

File: labsystem/src/Checkbox.test.js (hunk 1 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:181]: ok! fixed.

File: labsystem/src/Checkbox.test.js (hunk 2 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:201]: fixed

File: labsystem/src/Checkbox.test.js (hunk 3 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:217]: fixed

File: labsystem/src/Checkbox.test.js (hunk 4 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:251]: fixed

File: labsystem/src/Checkbox.test.js (hunk 5 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:261]: fixed

File: labsystem/src/Checkbox.test.js (hunk 6 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:283]: fixed

File: labsystem/src/Checkbox.test.js (hunk 7 above)
Comment (CavalcanteLucas) [reply labsystem/src/Checkbox.test.js:296]: fixed

**Review by lucianoratamero — APPROVED** [review]

**Pull request merged**